### DISCUSSION TITLE
More succinct docs when using JIT

### DISCUSSION THREAD

**Discussion by ghost** [tailwindlabs/tailwindcss#4093] [Ideas]
Example - I probably won't need this on every page when using JIT

![2021-04-13_081215](https://user-images.githubusercontent.com/11320080/114502308-3ff22b80-9c34-11eb-88c6-17d6c4130aff.png)

**Comment by simonswiss** [comment]
Hey!

JIT is currently still an "opt-in" mode, and not the default. We can't remove that sort of information that is still relevant for the default way Tailwind works, and managing two parallel sets of documentation would be a lot of work.

We could have a toggle for "jit mode vs default mode" for these, but realistically we'll probably update the documentation when `jit` becomes the default mode for Tailwind.

Moving this to our community discussions 👍

**Reply by ghost** [reply]
`We could have a toggle for "jit mode vs default mode" for these`
Yeah, I wanted something like this. Similar to how dark/light mode toggles save your preference in localStorage
